projects/application/src/app/pages/page-components/page-chip/page-chip-example.html

```diff
@@ -0,0 +1,80 @@
+<app-content>
+  <p><em>cb-chip</em> is made for using as a chip in your app.</p>
+
+  <h2>Single chip</h2>
+
+  <cb-chip>Default chip</cb-chip>
+
+  <h3>Sizes</h3>
+
+  <cb-chip size="small">Small chip</cb-chip>
+  <cb-chip size="medium">Medium chip</cb-chip>
+  <cb-chip size="large">Large chip</cb-chip>
+
+  <h3>With icons</h3>
+
+  <cb-chip size="small" icon="github">Small chip</cb-chip>
+  <cb-chip size="medium" icon="github">Medium chip</cb-chip>
+  <cb-chip size="large" icon="github">Large chip</cb-chip>
+
+  <h3>Colors</h3>
+
+  <cb-chip color="blue">Blue chip</cb-chip>
+  <cb-chip color="green">Green chip</cb-chip>
+  <cb-chip color="orange">Orange chip</cb-chip>
+  <cb-chip color="gray-light">Gray light chip</cb-chip>
+
+  <h3>Colors with background</h3>
+
+  <cb-chip color="blue" bg="true">Blue chip</cb-chip>
+  <cb-chip color="green" bg="true">Green chip</cb-chip>
+  <cb-chip color="orange" bg="true">Orange chip</cb-chip>
+  <cb-chip color="gray-light" bg="true">Gray light chip</cb-chip>
+
+  <h2>Chip list</h2>
+
+  <p><em>cb-chip-list</em> is made for create a wrapper to use more the one <em>cb-chip</em> and provide a spacements between items.</p>
```

Comment:
`is made for creating a wrapper to use more than one <em>cb-chip</em>. It provides spacements between items`.